docs/deploying.md

```diff
@@ -6,137 +6,125 @@ IGNORES-->
 
 # Deploying a registry server
 
-You obviously need to [install Docker](https://docs.docker.com/installation/) (remember you need **Docker version 1.6.0 or newer**).
+You obviously need to [install Docker version 1.6.0 or newer](https://docs.docker.com/installation/).
 
-## Getting started
+## Running on localhost
 
 Start your registry:
 
-     $ docker run -d -p 5000:5000 \
-     		--restart=always --name registry registry:2
+    docker run -d -p 5000:5000 --restart=always --name registry registry:2
 
-That's it.
+You can now use it with docker:
 
-You can now tag an image and push it:
+	# Get any image from the hub and tag it to point to your registry:
+    docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
 
-    $ docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
-    $ docker push localhost:5000/batman/ubuntu
+	# ... then push it to your registry:
+    docker push localhost:5000/batman/ubuntu
 
-Then pull it back:
+	# ... then pull it back from your registry:
+    docker pull localhost:5000/batman/ubuntu
 
-    $ docker pull localhost:5000/batman/ubuntu
+Now, stop your registry:
 
-## Where is my data?
+    docker stop registry && docker rm registry
 
-By default, your registry stores its data on the local filesystem, inside the container.
+## Storage
 
-In a production environment, it's highly recommended to use [another storage backend](https://github.com/docker/distribution/blob/master/docs/storagedrivers.md), by [configuring it](https://github.com/docker/distribution/blob/master/docs/configuration.md#storage).
+By default, your registry data is stored *inside the container* - thus it will be lost if you remove the container.
```

Comment:
I don't have the opportunity to do a full review right now, but as of https://github.com/docker/distribution/pull/615 there's now a `VOLUME` declaration for the data, so the registry data will end up in a volume.

Things to consider pointing out;

- Use `docker rm -v` to remove the volume together with the container (otherwise the volume gets "orphaned")
- User `volumes-from` when starting a new container to replace the old container during upgrade, so that the existing volume is attached to the new container and data is preserved, e.g.

```
docker rename registry old-registry
docker stop old-registry
docker run --volumes-from old-registry registry:2.xx
docker rm old-registry
```

Could be in a section "upgrading/updating"

^^ just some quick thoughts